data/index.html

```diff
@@ -133,40 +133,52 @@ <h5 class="text-center">USGS Topo Maps</h5>
+            <li>ArcGIS Online hosted feature services</li>
+            <li>SGID Open Data site</li>
+            <li>Raster Discovery web app</li>
+            <li>Discover basemap and imagery services</li>
+            <li>The AGRC Web API</li>
+        </ul>
         </p>
-    </div>
-    <div class="grid__col section alternate">
-    <h2>SGID Index</h2>
-        <div class="grid__col--span-all grid__col--centered grid__col--s-hidden" style="width:568px">
-            <a href="{% link data/sgid-index/index.html %}">
-                <img alt="SGID Index Table Search" class="text-center" src="{% link images/sgid-index.png %}">
-            </a>
-        </div>
-        <p class="readable">The SGID Index is a sortable and filterable table containing data items contained in our SGID. It is a great resource to quickly find GIS data and resources important to you. In the SGID Index, you will find the ISO category, the name of the resource, the agency who is the data steward, a short description and a link to a service endpoint if one is available. </p>
+
+        <p class="readable">The <a href="{% link data/sgid-index/index.html %}">SGID Index</a> is the master data index, regardless of where the data are stored or how they are served. It includes links to AGRC-maintained datasets as well as links to the authoritative sources for datasets created and served by other state agencies. Clicking on the <a href="#data-categories">category icons</a> on this page will take you to the associated category in the Index.</p>
         <p class="section-read-more">
-            <a href="{% link data/sgid-index/index.html %}" class="btn">SGID Index</a>
+            <a href="{% link data/sgid-index/index.html %}" class="btn">Search the SGID Index</a>
         </p>
     </div>
     <div class="grid__col section">
-        <h2>Raster Discovery and Download</h2>
-        <div class="grid__col--2-of-2 grid__col--centered grid__col--s-hidden" style="width:300px">
-            <img alt="SGID Index Table Search" class="text-center" src="{% link images/applications/raster_utah_gov.png %}">
-        </div>
-        <p class="readable">The Raster Discovery App allows users to get map-based results for all of the imagery, elevation, lidar, and scanned maps available from Utah's SGID. Select the types of products you are interested in. Next, define your area of interest. The Raster Discovery App will search the product indices and produce a resulting list of items that intersected your area of interest. Finally, download the data! </p>
-        <p class="section-read-more">
-            <a href="https://raster.utah.gov" class="btn">Raster App</a>
+        <h2>Choosing a Data Source</h2>
+        <p class="readable">With so many ways to get SGID Data, it's easy to get confused when trying to decide what to use. This guide will help you understand the best ways to access the data based on where you're trying to use it.</p>
```

Comment:
**suggestion**: we have smart users. let's not lead them to believe it is confusing :)
**question**: why is data capitalized? I'm not very good at english.
```suggestion
        <p class="readable">With so many ways to get SGID Data, we created a guide to help you understand the best ways to access the data based on your use case.</p>
```